### PR TITLE
Comments to the graph and measure in create_element

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

File: braph2genesis/genesis/create_Element.m [hunk 1]
```diff
@@ -1099,37 +1099,89 @@ function generate_header_graph()
...
                 ~(numel(selfconnectivity) == 1 && isempty(selfconnectivity{1})) || ...
                 ~(numel(negativity) == 1 && isempty(negativity{1}))
-            g(1, 'methods (Static) %% graph methods')
+            g(1, 'methods (Static) %% graph methods')                
                 if ~isempty(ensemble)
                     g(2, 'function bool = is_ensemble()')
+                        gs(3, {
+                            ['%IS_ENSEMBLE returns true if ' descriptive_name ' is an ensamble.']
+                            '%'
+                            ['% BOOL = ' descriptive_name '.IS_ENSEMBLE() returns true if it is an ensamble graph.']
```
Comment (annacanal) [inline braph2genesis/genesis/create_Element.m:1108]: ensamble -> ensemble

File: braph2genesis/genesis/create_Element.m [hunk 2]
```diff
@@ -1099,37 +1099,89 @@ function generate_header_graph()
...
                     g(2, 'end')
                 end
                 if ~(numel(selfconnectivity) == 1 && isempty(selfconnectivity{1}))
                     g(2, 'function selfconnectivity = getSelfConnectivityType(varargin)')
+                        gs(3, {
+                            ['%GETSELFCONNECTIVITYTYPE returns ' descriptive_name ' type of graph self connectivity.']
+                            '%'
+                            ['% SELFCONNECTIVITY = ' descriptive_name '.GETSELFCONNECTIVITYTYPE() returns the type of graph self connnectivity.']
+                            '% Graphs can be self connected or non self connected.'
+                            '%'
```
Comment (annacanal) [inline braph2genesis/genesis/create_Element.m:1166]: **self-connectivity**
Graphs can be self-connected or non self-connected.'

File: braph2genesis/genesis/create_Element.m [hunk 3]
```diff
@@ -1099,37 +1099,89 @@ function generate_header_graph()
...
                 end
                 if ~(numel(negativity) == 1 && isempty(negativity{1}))
                     g(2, 'function negativity = getNegativityType(varargin)')
+                        gs(3, {
+                                ['%GETNEGATIVITYTYPE returns ' descriptive_name ' type of graph negativity.']
+                                '%'
+                                ['% NEGATIVITY = ' descriptive_name '.GETNEGATIVITYTYPE() returns the type of graph negativity.']
+                                '% Graphs can be negative or non negative.'
+                                '%'
+                                '% See also is_ensemble, getGraphType, getConnectivityType, getDirectionalityType, getNegativityType.'
```
Comment (annacanal) [inline braph2genesis/genesis/create_Element.m:1182]: getNegativityType ->  **getSelfConnectivityType**

File: braph2genesis/genesis/create_Element.m [hunk 4]
```diff
@@ -1147,21 +1199,56 @@ function generate_header_measure()
             g(1, 'methods (Static) %% graph methods')
                 if ~isempty(shape)
                     g(2, 'function shape = getMeasureShape()')
+                        gs(3, {
+                            ['%GETMEASURESHAPE returns ' descriptive_name ' measure graph shape.']
+                            '%'
+                            ['% SHAPE = ' descriptive_name '.GETMEASURESHAPE() returns the measures graph shape.']
+                            '%   Shape can be global, nodal, binodal.'
```
Comment (annacanal) [inline braph2genesis/genesis/create_Element.m:1206]: Shape can be global, nodal or binodal.

File: braph2genesis/genesis/create_Element.m [hunk 5]
```diff
@@ -1147,21 +1199,56 @@ function generate_header_measure()
...
                         g(3, shape)
                     g(2, 'end')
                 end
                 if ~isempty(scope)
                     g(2, 'function scope = getMeasureScope()')
+                        gs(3, {
+                            ['%GETMEASURESCOPE returns ' descriptive_name ' measure layer scope.']
+                            '%'
+                            ['% SCOPE = ' descriptive_name '.GETMEASURESCOPE() returns the measures layer scope.']
+                            '%   Scope can be unilayer, superglobal or bilayer.'
```
Comment (annacanal) [inline braph2genesis/genesis/create_Element.m:1220]: I would write in the following order, as it appears in line 42 of this file create_Element
Scope can be superglobal, unilayer or bilayer.

**Review by annacanal — CHANGES_REQUESTED** [review]
small things

**Review by c-yuwei — APPROVED** [review]

**Review by annacanal — APPROVED** [review]

File: braph2genesis/genesis/create_Element.m [hunk 6]
```diff
@@ -1099,37 +1099,89 @@ function generate_header_graph()
...
                 ~(numel(selfconnectivity) == 1 && isempty(selfconnectivity{1})) || ...
                 ~(numel(negativity) == 1 && isempty(negativity{1}))
-            g(1, 'methods (Static) %% graph methods')
+            g(1, 'methods (Static) %% graph methods')                
                 if ~isempty(ensemble)
                     g(2, 'function bool = is_ensemble()')
+                        gs(3, {
+                            ['%IS_ENSEMBLE returns true if ' descriptive_name ' is an ensemble.']
+                            '%'
+                            ['% BOOL = ' descriptive_name '.IS_ENSEMBLE() returns true if it is an ensemble graph.']
```
Comment (giovannivolpe) [inline braph2genesis/genesis/create_Element.m:1108]: make it concrete with two cases: one for true and one for false

File: braph2genesis/genesis/create_Element.m [hunk 7]
```diff
@@ -1099,37 +1099,89 @@ function generate_header_graph()
...
                     g(2, 'end')
                 end
                 if ~(numel(connectivity) == 1 && isempty(connectivity{1}))
                     g(2, 'function connectivity = getConnectivityType(varargin)')
+                        gs(3, {
+                            ['%GETCONNECTIVITYTYPE returns ' descriptive_name ' type of graph connectivity.']
+                            '%'
+                            ['% CONNECTIVITY = ' descriptive_name '.GETCONNECTIVITYTYPE() returns the type of graph connectivity.']
+                            '% Graphs can have binary or weighted connectivity.' 
+                            '%'
```
Comment (giovannivolpe) [inline braph2genesis/genesis/create_Element.m:1136]: Add case with number of layers in varaging, also in methods below

**Review by giovannivolpe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**